diff --git a/CMakeLists.txt b/CMakeLists.txt
index 089a20b..14be860 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -38,6 +38,8 @@ cmake_minimum_required(VERSION 2.8.12...3.12)
 
 PROJECT(libfreenect)
 
+SET(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
+
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/cmake_modules/")
 
 # Find the host operating system and architecture
